.vsts-dotnet-ci.yml

```diff
@@ -0,0 +1,21 @@
+resources:
+  containers:
+  - container: centos7
+    image: microsoft/dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416
```

Comment:
>dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416 [](start = 21, length = 57)

does this docker image contains libgdiplus?

Reply:
Yes it does: 

![image](https://user-images.githubusercontent.com/22899328/43091828-0c6b34e4-8e60-11e8-87db-30ad9b2814f9.png)

https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/56

Reply:
[https://ci2.dot.net/job/Private/job/dotnet_machinelearning/job/master/job/linux_release_prtest/1240/testReport/junit/(root)/(empty)/Microsoft_ML_Tests_ImageTests_TestBackAndForthConversion/](https://ci2.dot.net/job/Private/job/dotnet_machinelearning/job/master/job/linux_release_prtest/1240/testReport/junit/(root)/(empty)/Microsoft_ML_Tests_ImageTests_TestBackAndForthConversion/)
I've merge master into my image branch, but it still failing on linux.

---
In reply to: [204483166](https://github.com/dotnet/machinelearning/pull/566#discussion_r204483166) [](ancestors = 204483166)

Reply:
The tests that you added passed in my machine when using that docker image. I just need to dig why they are failing in VSTS, maybe not picking the right docker image. I'll update you.